src/main/java/database/Database.java

```diff
@@ -0,0 +1,170 @@
+package database;
+
+import java.io.File;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import config.Config;
+
+/** A class that handles generic database operations. */
+public class Database {
+    private static Connection conn;
+
+    /**
+     * Connect to the database.
+     * @throws SQLException
+     *             if a database error occurs
```

Comment:
Leave an empty line between description and parameter section.

Reply:
![image](https://github.com/nus-cs2103-AY2324S2/ip/assets/81742181/1ee5bd57-9ed6-4ea3-8ff1-12258908f930)

It seems that the checkstyle xml file did not like this 💀

Reply:
Never mind i figured out the issue.